internal/venice-common/src/main/java/com/linkedin/venice/meta/Partition.java

```diff
@@ -26,19 +33,33 @@
 public class Partition {
   private static final Logger LOGGER = LogManager.getLogger(Partition.class);
   /**
-   * ID of partition. One of the number between [0 ~ total number of partition)
+   * ID of partition. One of the number between [0, total number of partition[
```

Comment:
IMO original syntax was correct. It is half-closed interval.

`[0, total number of partition) --> [a, b) ==> {x belons to N | a <= x && x < b}`

Reply:
Seems like both syntaxes are correct... I didn't know about the parenthesis syntax for exclusion!

https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints